pkg/client/html_templates.go

```diff
@@ -447,6 +450,8 @@ func (c *Client) renderTemplate(
 		ClientInfo:  clientInfo,
 		Disks:       c.getDisks(ctx),
 		Headers:     req.Header,
+		Tunnel:      c.tunnel.Config,
+		PoolStats:   c.tunnel.PoolStats(),
```

Comment:
Instead of pass in pieces of `c.tunnel`, pass in the whole thing so we can `if` statement it in the template. Sometimes `c.tunnel` is `nil`, and line 454 causes a crash.